services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/Persistence.kt

```diff
@@ -0,0 +1,43 @@
+package fi.hel.haitaton.hanke.tormaystarkastelu
+
+import fi.hel.haitaton.hanke.HankeEntity
+import org.springframework.data.jpa.repository.JpaRepository
+import java.time.LocalDateTime
+import javax.persistence.Embedded
+import javax.persistence.Entity
+import javax.persistence.FetchType
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Entity
+@Table(name = "tormaystarkastelutulos")
+class TormaystarkasteluTulosEntity {
+    @Embedded
+    var liikennehaitta: LiikennehaittaIndeksiType? = null
+    var perus: Float? = null
+    var pyoraily: Float? = null
+    var joukkoliikenne: Float? = null
+
+    var tila: TormaystarkasteluTulosTila? = null
+    var createdAt: LocalDateTime? = null
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    var id: Int? = null
+
+    @ManyToOne(optional = false, fetch = FetchType.LAZY)
```

Comment:
That optional = false is something that might be useful for some other relations we already have. Read that without it, it defaults to true, in which case certain queries may end up doing unnecessary join stuff..